packages/theming/apple-theme/src/appleTheme.macos.ts

```diff
@@ -15,13 +13,33 @@ export function appleSpacing(): Spacing {
 }
 
 export const appleComponents = {
+  // These values correspond to the "Large"  Button Size values of the FluentUI Apple Button
   Button: {
     tokens: {
-      borderRadius: 5,
+      borderRadius: 6,
```

Comment:
Did you just pull this from an actual NSButton you tested yourself? Wondering why it was wrong in the first place

Reply:
I pulled these from the FluentUI Apple macOS Button, by looking at what we set it as in Swift. Those come from a Figma (I assume).
It was wrong because I wrote it wrong the first time.